### PR TITLE
Added the link to the blog to the Machine learning projects with sour…

### LINKED ISSUE #1700: [Add] <Top 100+ Machine Learning Projects for 2023 [with Source Code]>
### Site URL

https://www.geeksforgeeks.org/machine-learning-projects/

### Description of the above provided site

In this article, you’ll find the top 100+ latest projects and Ideas on Machine Learning which are beneficial for both beginners as well experienced professionals. These projects are specially made for the students that start their journey in Machine Learning and Data Science.

### Category

Artificial Intelligence

### Sub Category

Machine learning

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by digpatel123**
Closes #1700

### CODE REVIEW

File: database/resources/blogs.json [hunk 1]
```diff
@@ -152,5 +152,12 @@
     "url": "https://www.notion.so/personal",
     "category": "resources",
     "subcategory": "blogs"
+  },
+  {
+    "name": "Top 100+ Machine Learning Projects for 2023 [with Source Code]",
+    "description": "In this article, you’ll find the top 100+ latest projects and Ideas on Machine Learning which are beneficial for both beginners as well experienced professionals. These projects are specially made for the students that start their journey in Machine Learning and Data Science.",
```
Comment (CBID2) [inline database/resources/blogs.json:158]:
```suggestion
    "description": "In this article, you’ll find the top 100+ latest projects and ideas on Machine Learning,which are beneficial for both beginners and experienced professionals. These projects are also helpful for the students looking to start their journey in Machine Learning and Data Science.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @digpatel123! :) I made a suggestion on how you can improve your description

File: database/resources/blogs.json [hunk 2]
```diff
@@ -152,5 +152,12 @@
     "url": "https://www.notion.so/personal",
     "category": "resources",
     "subcategory": "blogs"
+  },
+  {
+    "name": "Top 100+ Machine Learning Projects for 2023 [with Source Code]",
+    "description": "In this article, you’ll find the top 100+ latest projects and ideas on Machine Learning,which are beneficial for both beginners and experienced professionals. These projects are also helpful for the students looking to start their journey in Machine Learning and Data Science.",
```
Comment (CBID2) [inline database/resources/blogs.json:158]:
```suggestion
    "description": "In this article, you’ll find the top 100+ latest projects and ideas on Machine Learning,which are beneficial for both beginners and experienced professionals. These projects are also helpful for  students looking to start their journey in Machine Learning and Data Science.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
I gave more feedback! :) Keep it up

**Review by CBID2 — APPROVED** [review]
Well done @digpatel123! :) Now, you just need another approval check! :)

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**